src/Core/Context/CurrentContext.cs

```diff
@@ -507,6 +510,11 @@ public bool AccessSecretsManager(Guid orgId)
         return Providers;
     }
 
+    public CurrentContextOrganization GetOrganization(Guid orgId)
+    {
+        return Organizations.Find(o => o.Id == orgId);
+    }
+
```

Comment:
@shane-melton pointed out to me that `CurrentContext` already has an interface for this, `OrganizationMembershipAsync`. Let's use that instead of adding our own. (I'll also update my own PR to match)

Reply:
Actually I just realised that the existing method gets _all_ organizations rather than a specific org. I still like adding an interface for this rather than having to `Find` the org yourself every time.

Maybe this should just become an async method which is a wrapper over `OrganizationMembershipAsync`?

Reply:
This gets really messy because the `OrganizationMembershipAsync` methods requires a reference to the `OrganizationUserRepository`. This poses problems for any usages within the current context itself.  Shane and I talked a week or two ago about this and both came to the conclusion that `Organizations` shouldn't ever be `null` since its initialized when the context is set via the `claimsDict`. Do you know of an instance where the `Organizations` property won't be set?